app/src/processing/app/EditorConsole.java

```diff
@@ -110,18 +113,86 @@ public EditorConsole() {
     setMinimumSize(new Dimension(100, (height * lines)));
 
     EditorConsole.init(stdOutStyle, System.out, stdErrStyle, System.err);
+
+    // Add "CTRL scroll" hotkey for font size adjustment.
+    consoleTextPane.addMouseWheelListener((MouseWheelEvent e) -> {
+      if (e.isControlDown()) {
+        if (e.getWheelRotation() < 0) {
+          base.handleFontSizeChange(1);
+        } else {
+          base.handleFontSizeChange(-1);
+        }
+      } else {
+        e.getComponent().getParent().dispatchEvent(e);
+      }
+    });
```

Comment:
I see this MouseWheelEvent repeated around, may this be factored out?

Reply:
This crossed my mind already, I will look into this.

Reply:
Done in 2d593c4.